.rubocop.yml

```diff
@@ -55,6 +55,12 @@ Style/HashSyntax:
 Rails/ActionOrder:
   Enabled: false
 
+Metrics/ClassLength:
+  Exclude:
+    - "test/**/*.rb"
```

Comment:
why was this necessary - is there a good reason we don't want the tests validating too?

Reply:
Ugh robocop loves to complain about everything. This is a test that your class is getting too long, which is common in minitest, so I disabled it for the minitest directory.

(rubocop was complaining that the PartnerTest class was too big)

Reply:
I'm not sure whether this is a good idea or not! @aaaaargZombies @katjam do we want rubocop in our test suite as well? It seems to me like there could be bad knock on effects of just taking it out, don't want us to start committing lots of messy code just cos it's in tests, but also, tests take up so much of out time as it is. Could we not just make another partner test for this specific behaviour if it's making the original too big?

Reply:
This is just disabled for only one test, not the entire rubocop bit at all.

Reply:
ohhhh I'm sorry i missed that this is specifically for `Metrics/ClassLength:` - I thought you were just excluding all tests from rubocop! In that case is probably fine